components/climate/climate_ir.rst

```diff
@@ -396,64 +335,61 @@ Configuration variables:
+    ``panasonic_jke``, ``panasonic_lke``, ``panasonic_nke``, ``r51m``, ``samsung_aqv``, ``samsung_aqv12msan``, ``samsung_fjm``, ``sharp``,
+    ``toshiba``, ``toshiba_daiseikai``, ``vaillantvai8``, ``zhjg01``, ``zhlt01``
+- **horizontal_default** (**Required**, string): What to default to when the AC unit's horizontal direction is *not* set to swing. Options are: ``left``, ``mleft``, ``middle``, ``mright``, ``right``, ``auto``
+- **vertical_default** (**Required**, string): What to default to when the AC unit's vertical direction is *not* set to swing. Options are: ``down``, ``mdown``, ``middle``, ``mup``, ``up``, ``auto``
+- **max_temperature** (**Required**, float): The maximum temperature that the AC unit supports being set to.
+- **min_temperature** (**Required**, float): The minimum temperature that the AC unit supports being set to.
+- **sensor** (*Optional*, :ref:`config-id`): The sensor that is used to measure the ambient temperature.
 
-Configuration variables:
+.. note::
 
-- **model** (*Required*, string): GREE has a few different protocols depending on model. One of these will work for you.
+    The ``greeyac`` protocol in ``heatpumpir`` supports a feature Gree calls "I-Feel". The handheld remote control
+    has a built-in temperature sensor and it will periodically transmit the temperature from this sensor to the
+    AC unit. If a ``sensor`` is provided in the configuration with this model, the sensor's temperature will be
+    transmitted to the ``greeyac`` device in the same manner as the original remote controller. How often the
+    temperature is transmitted is determined by the ``update_interval`` assigned to the ``sensor``. Note that
+    ``update_interval`` must be less than 10 minutes or the ``greeyac`` device will revert to using its own
+    internal temperature sensor; a value of 2 minutes seems to work well. See :doc:`/components/sensor/index`
+    for more information.
 
-  - ``generic``
-  - ``yan``
-  - ``yaa``
-  - ``yac``
-  - ``yac1fb9``
+.. _delonghi_ir:
 
+.. note::
 
-.. code-block:: yaml
+    The ``delonghi`` climate currently supports the protocol used by some Delonghi portable units, known working with Delonghi PAC WE 120HP.
 
-    # Example configuration entry
-    climate:
-      - platform: gree
-        name: "AC"
-        sensor: room_temperature
-        model: yan
+.. _daikin_arc:
 
-.. _zhlt01:
+.. note::
 
+    The Daikin ARC remotes (``daikin_arc`` climate, ``daikin_arc417``, ``daikin_arc480`` protocols) are used by the japanese model of Daikin.
 
-``zhlt01`` Climate
----------------------
+.. _zhlt01:
 
-ZH/LT-01 is a remote control that is used with many locally branded split airconditioners.
-Supported brands include:
-
-- Eurom
-- Chigo
-- Tristar
-- Tecnomaster
-- Elgin
-- Geant
-- Tekno
-- Topair
-- Proma
-- Sumikura
-- JBS
-- Turbo Air
-- Nakatomy
-- Celestial Air
-- Ager
-- Blueway
-- Airlux
-
-No additional configuration is required for this model.
+.. note::
```

Comment:
We should leave the heading there so they populate correctly in the sidebar. Really I think these sections should stay in the same order as the table at the top please.

```suggestion
``zhlt01`` Climate
---------------------
```